tests/tests_integration/test_api/test_data_modeling/test_views.py

```diff
@@ -79,6 +79,17 @@ def test_delete_non_existent(self, cognite_client: CogniteClient, integration_te
             == []
         )
 
+    def test_retrieve_without_inherited_properties(self, cognite_client: CogniteClient, cdf_views: ViewList) -> None:
+        assert len(cdf_views) >= 1, "Please add at least one view to the test environment"
```

Comment:
Let's put this assertion in the fixture instead? It will be entirely useless if doesn't return any data. It would be even better to retrieve them by ids there and raise if any are missing, then the fixture becomes self-documenting aswell.